scripts/countme-weekly-totals-ui.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+DISTROS=('Red Hat Enterprise'
+	 'CentOS Linux'
+	 'CentOS Stream'
+	 'Oracle Linux'
+	 'AlmaLinux'
+	 'Rocky'
+	 );
+WEEKS=2
+
+if [ "x$1" = "x--help" -o "x$1" = "xhelp" ]; then
+	echo "$0: [WEEKS]"
+	echo "  Show CSV data in a more usable form for N weeks ago. Default: 2"
+	echo "  0/1 works but may be incomplete."
+    exit 0
+fi
+
+if [ "x$1" != "x" ]; then
+	WEEKS="$1"
+fi
+
+DAYS=$((7 * $WEEKS))
+DATE=$( date -d "last monday - ${DAYS} days" -I );
+
+ARCHES=('x86_64' 'aarch64' 'ppc64le' 's390x');
+RELEASES=('epel-8' 'epel-9');
+RELEASE3=('centos-baseos-9-stream');
```

Comment:
Were these on a single line before? Anyway, let’s remove the trailing semicolons.